src/lib/Content/View/Builder/ContentEditViewBuilder.php

```diff
@@ -117,7 +117,9 @@ public function buildView(array $parameters)
             'language' => $language,
             'content_type' => $contentType,
             'form' => $form->createView(),
-            'grouped_fields' => $this->getGroupedFields($form),
+            'grouped_fields' => $this->groupedContentFormFieldsProvider->getGroupedFields(
+                $form->get('fieldsData')->all()
+            ),
```

Comment:
TBH while it's a change to initial implementation, I believe that probably `AbstractType::buildView()` from the Form component should have been used instead for grouping data for the form. After all, how fields are grouped should be part of the form itself, not part of a view.

Just a note though. Optional.